CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+5.1.4 / WIP
```

Comment:
Since this changes how the endpoint behaves, a minor version should be updated (5.2.0)